### DISCUSSION TITLE
Completely remove dashboard

### DISCUSSION THREAD

**Discussion by plakhin** [filamentphp/filament#1046] [Help] [answered]
I want to completely remove dashboard page. If I just remove it's registration from config, I'm getting 404 error when visiting `/admin` (once authorized). How can I make the index page of first Resource be a default page (`/admin`)?

**Comment by plakhin** [comment, marked as answer]
Currently I've just set permanent redirect in routes from `/admin` to `/admin/users`

**Comment by aurawindsurfing** [comment]
Easier way would be to edit filament config and deregister Dashboard page:

```
'pages' => [
        'namespace' => 'App\\Filament\\Pages',
        'path' => app_path('Filament/Pages'),
        'register' => [
            //Pages\Dashboard::class,
        ],
    ],
```

**Reply by aurawindsurfing** [reply]
Actually I realised it is incorrect answer, we would need to change redirect after login in order for it to work
